classes/ValidFormBuilder/ValidForm.php

```diff
@@ -1847,13 +1847,16 @@ protected function __toJS($strCustomJs = "", $arrInitArguments = array(), $blnRa
      * @param $intDynamicCount
      * @return bool|true
      */
-    protected function elementShouldDisplay(Base $objElement, $intDynamicCount = 0)
+    protected function elementShouldDisplay(Base $objElement)
```

Comment:
I would not change the method signature. There is already a default value of 0 set, just leave as is.